src/VisualStudio/Core/Test.Next/project.json

```diff
@@ -2,18 +2,8 @@
   "dependencies": {
     "Newtonsoft.Json": "8.0.3",
     "Moq": "4.2.1402.2112",
```

Comment:
by the way, which dlls end up in binary directory were un-deterministic for me without explicit references.